scripts/set-alias-page.py

```diff
@@ -8,8 +8,8 @@
 isn't suggested to use the sync option. If used, only stage changes
 and commit verified changes for your language by using -l LANGUAGE.
 
-Note: If there is a symlink error when using the stage flag remove the `pages.en`
-directory temporarily and try executing it again.
+Note: If the current directory or one of its parents is called "tldr", the script will assume it is the tldr root, i.e., the directory that contains a clone of https://github.com/tldr-pages/tldr
+If you aren't, the script will use TLDR_ROOT as the tldr root. Also, ensure 'git' is available.
```

Comment:
```suggestion
If the script doesn't find it in the current path, the environment variable TLDR_ROOT will be used as the tldr root. Also, ensure 'git' is available.
```
I think it's worth to clarify that "TLDR_ROOT" is an environment variable.